include/sound/soc-acpi-intel-match.h

```diff
@@ -51,6 +51,9 @@ extern struct snd_soc_acpi_mach snd_soc_acpi_intel_ptl_sdw_machines[];
  * generic table used for HDA codec-based platforms, possibly with
  * additional ACPI-enumerated codecs
  */
+#define SND_SOC_ACPI_INTEL_HDA_MACH_SKL	0
+#define SND_SOC_ACPI_INTEL_HDA_MACH_SOF	1
```

Comment:
do we need to do this?

Most of the SKL/KBL platforms with the DSP are supposed to use the AVS driver now.

Rather than deal with differences with an obsolete driver, maybe we should remove support for HDaudio in SKL?

@crojewsk what do you think?

Reply:
Sorry for the delay. I'll take a look today and respond either today or tomorrow. Thank you @plbossart for informing me about this.

Reply:
ACK for ignoring skylake-driver here.